utils/json/Cargo.toml

```diff
@@ -20,3 +20,10 @@ multihash = { workspace = true, default-features = false, features = [
 ] }
 num-bigint.workspace = true
 serde = { workspace = true, features = ["derive"] }
+
+[dev-dependencies]
+forest_test_utils           = { workspace = true, features = ["test_constructors"] }
```

Comment:
`test_constructors` is not strictly necessary but that feature will be removed soon in any case so it doesn't hurt to leave it.